03-Node-npm-modules/movie.js

```diff
@@ -0,0 +1,31 @@
+var Director = require('./director.js');
+
+var Movie = function () {
+  this.setName(name);
+  this.director = new Director();
+};
+
+Movie.prototype = {
+	constructor: Movie,
```

Comment:
You don't need to specify the constructor if you are not extending another class/constructor.